### PR TITLE
adding info about option to open target in new tab

### PR DESCRIPTION
**Pull request by emuus**
information added for examples both in header and footer pages

### CODE REVIEW

File: admin/en/source/pages/rebranding/custo-footer.rst [hunk 1]
```diff
@@ -39,11 +39,14 @@ The following example shows how to add a link to twitter:
 	           <c:title>twitter</c:title>
 	           <c:icon>images/common/icons/twitter.svg</c:icon>
 	           <c:link>https:twitter.com/usharesoft</c:link>
+	           <c:target>blank</c:target>
 		</c:linkItem>
 	</c:footer>
 
 You can add as many links as you like by adding a ``linkItem`` for each link.
 
+If ``<c:target>blank</c:target>`` or ``<c:target>_blank</c:target>`` are used with ``<c:linkItem>`` the page will open if a new tab.
```
Comment (jgweir) [inline admin/en/source/pages/rebranding/custo-footer.rst]: do you not mean "open IN a new tab" ?

File: admin/en/source/pages/rebranding/custo-footer.rst [hunk 2]
```diff
@@ -58,10 +61,13 @@ To modify the Terms of Use or Privacy Policy, go to the sections under <c:footer
...
+	           <c:target>blank</c:target>
 		</c:linkItem>
 		<c:linkItem>
 	           <c:title>privacy policy</c:title>
 	           <c:link>https//www.usharesoft.com/about/privacy-policy.html</c:link>
+	           <c:target>blank</c:target>
 		</c:linkItem>
 	</c:footer>
 
+If ``<c:target>blank</c:target>`` or ``<c:target>_blank</c:target>`` are used with ``<c:linkItem>`` the page will open if a new tab.
```
Comment (jgweir) [inline admin/en/source/pages/rebranding/custo-footer.rst]: same comment as above

File: admin/en/source/pages/rebranding/custo-header.rst [hunk 3]
```diff
@@ -55,5 +55,8 @@ The following example shows how to add a link to a blog under the ``<c:header>``
         <c:linkItem>
     		<c:title>Blog</c:title>
         	<c:link>https://blog.usharesoft.com/</c:link>
+        	<c:target>blank</c:target>
     	</c:linkItem>
     </c:header>
+
+If ``<c:target>blank</c:target>`` or ``<c:target>_blank</c:target>`` are used with ``<c:linkItem>`` the page will open if a new tab.
```
Comment (jgweir) [inline admin/en/source/pages/rebranding/custo-header.rst]: same as above

**Comment by emuus** [comment]
Good catch. Changes done

**Pull request merged**